ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/model/jsonrpc/EeaSendTransaction.java

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.jsonrpcproxy.model.jsonrpc;
+
+import static java.util.Collections.singletonList;
+
+import tech.pegasys.ethsigner.jsonrpcproxy.model.jsonrpc.PrivateTransaction.PrivateTransactionBuilder;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import io.vertx.core.json.Json;
+import org.web3j.protocol.core.Request;
+import org.web3j.protocol.core.methods.response.EthSendTransaction;
+
+public class EeaSendTransaction {
+  private static final String RESTRICTED = "restricted";
+  private static final int DEFAULT_ID = 77;
+  private static final String UNLOCKED_ACCOUNT = "0x7577919ae5df4941180eac211965f275cdce314d";
+  private static final String PRIVATE_FROM = "ZlapEsl9qDLPy/e88+/6yvCUEVIvH83y0N4A6wHuKXI=";
+  private static final List<String> PRIVATE_FOR =
+      singletonList("GV8m0VZAccYGAAYMBuYQtKEj0XtpXeaw2APcoBmtA2w=");
+
+  public String withGas(final String gas) {
+    return replaceParameter("gas", gas, request());
+  }
+
+  public String withGasPrice(final String gasPrice) {
+    return replaceParameter("gasPrice", gasPrice, request());
+  }
+
+  public String withValue(final String value) {
+    return replaceParameter("value", value, request());
+  }
+
+  public String withNonce(final String nonce) {
+    return replaceParameter("nonce", nonce, request());
+  }
+
+  public String withSender(final String receiver) {
+    return replaceParameter("from", receiver, request());
+  }
+
+  public String withReceiver(final String sender) {
+    return replaceParameter("to", sender, request());
+  }
+
+  public String missingSender() {
+    final PrivateTransaction transaction =
+        new PrivateTransactionBuilder()
+            .withNonce("0xe04d296d2460cfb8472af2c5fd05b5a214109c25688d3704aed5484f9a7792f2")
+            .withGasPrice("0x9184e72a000")
+            .withGas("0x76c0")
+            .withTo("0xd46e8dd67c5d32be8058bb8eb970870f07244567")
+            .withValue("0x0")
+            .withData(
+                "0xd46e8dd67c5d32be8d46e8dd67c5d32be8058bb8eb970870f072445675058bb8eb970870f072445675")
+            .withPrivateFrom(PRIVATE_FROM)
```

Comment:
nit; this nonce string could be pull into a `DEFAULT_DATA` constant. Cleaner code / quicker readability (especially if another data is ever used)

Reply:
done